README

```diff
@@ -0,0 +1,55 @@
+# About Check
```

Comment:
Oops - I see what you mean - I'll go and clean those out - not sure how that happened - wrong git add option I suspect.